index.html

```diff
@@ -2,30 +2,191 @@
 
 <html lang="en">
 <head>
-  <meta charset="utf-8">
+    <meta charset="utf-8">
 
-  <title>Digital Resume</title>
+    <title>Digital Resume</title>
 
-  <link rel="stylesheet" href="css/bootstrap.min.css">
-  <link rel="stylesheet" href="css/index.css">
+    <link rel="stylesheet" href="css/bootstrap.min.css">
+    <link rel="stylesheet" href="css/index.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
 
-  <!--[if lt IE 9]>
+    <!--[if lt IE 9]>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
-  <![endif]-->
+    <![endif]-->
 </head>
 
 <body>
 
-    <div class="container">
-      <div class="row">
-          <div class="col-sm-12">
-              <h1>Start Here!</h1>
-          </div>
-      </div>
-    </div><!-- container -->
-
-    <script src="js/jquery-3.3.1.min.js"></script>
-    <script src="js/bootstrap.bundle.min.js"></script>
-    <script src="js/index.js"></script>
+<div class="container">
+    <div class="row justify-content-end">
+        <nav class="nav">
+            <!--<a class="nav-link active" href="#">Home</a>-->
+            <a class="nav-link" href="#">About</a>
+            <a class="nav-link" href="#">Portfolio</a>
+            <a class="nav-link" href="#">Contact</a>
+        </nav>
+    </div>
+    <div class="row justify-content-center">
+        <header>
+            <h1>Harrison Brock</h1>
+        </header>
+    </div>
+    <h4>Projects</h4>
+    <div class="row justify-content-center">
+        <div class="col-md-6">
+            <div class="card" style="width: 18rem;">
```

Comment:
You have a weird spacing issue with your images, the reason is because of this implementation of `style="width: 18rem;`. You are overwriting some properties that bootstrap has. When I remove this, the images are extremely large so you may want to fix the sizing on that. If you want to keep the card size, you can keep your style and add margin left and right to somehow align your item. Other than that, I like what you have!